boilerplates/react-union-boilerplate-liferay-basic/hero-portlet/src/main/java/eu/reactunion/boilerplate/portlet/HeroReactWidgetSupport.java

```diff
@@ -0,0 +1,68 @@
+package eu.reactunion.boilerplate.portlet;
+
+import javax.portlet.RenderRequest;
+
+import com.liferay.portal.kernel.module.configuration.ConfigurationException;
+import com.liferay.portal.kernel.theme.PortletDisplay;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.WebKeys;
+import eu.reactunion.boilerplate.configuration.HeroConfiguration;
+import eu.reactunion.boilerplate.portlet.data.InitData;
+import eu.reactunion.boilerplate.portlet.data.Textation;
+import org.osgi.service.component.annotations.Component;
+
+import eu.lundegaard.reactunion.support.ReactWidgetSupport;
+
+/**
+ * @author Roman Srom (roman.srom@lundegaard.eu)
+ */
+@Component(
+        immediate = true,
+        service = HeroReactWidgetSupport.class
+)
+public class HeroReactWidgetSupport extends ReactWidgetSupport {
+
+//    @Override
+//    public Map<String, Object> getWidgetsInitData(RenderRequest request) {
```

Comment:
No commented blocks of code in the codebase, unless meant to be uncommented for some special usage, please

Reply:
hero-portlet folder was removed completely because it was moved to liferay folder. Bug caused by old code and master merge.